### ISSUE TITLE
[dev-server-storybook] Addons requirements

### ISSUE THREAD

**Issue by vlaraort** [modernweb-dev/web#1341]
Hey👋

I have a custom storybook addon and I am unable to load it in the dev-server-storybook, while it works fine in a regular storybook without the @web implementation

This is the addon: [https://github.com/vlaraort/addon-screen-reader](https://github.com/vlaraort/addon-screen-reader)

I know that currently `It's not possible to install other addons at the moment.`, but as there are some addons that are being used, I suppose that only addons written/exported in a concrete way can be used. As I am the dev of the addon, I was wondering if there is something that I can change to make the addon compatible with dev-server-storybook, and if you can point me into the correct path 🙈

Regards, and keep the good work! 🎉

**Comment by LarsDenBakker** [comment]
We don't have any docs for this yet, but the plugin would have to import all it's dependencies from the storybook prebuilt package https://github.com/modernweb-dev/storybook-prebuilt

**Comment by vlaraort** [comment]
Thank you for the response!

But I don't think that I get it totally sorry, you mean that I should add storybook-prebuild as a dependency of my module, and import all the `@storybook/...`, `react` and `react-dom` from there? 

Also, looks like that I should export the component as ES6 module, instead of buiding it to CommonJS, right?

Last question, there is any custom addon that uses this approach to take it as reference?

Thank you so much Lars :)

**Comment by LarsDenBakker** [comment]
Yes that's correct, the prebuilt ships an es module variant and any addon would need to ship one as well. 

There is no example addon for this right now, and no docs, I realize that makes it a bit hard :) The goal for the prebuild is primarily to ship a build of storybook that people can use as is, if people need a lot of customization it might be better to use regular storybook as is.

**Comment by vlaraort** [comment]
Yup, I see, anyway I work on an orange company (😉)and we prefer to stick on your awesome implementation, and this addon is something that I want to be added to some projects, so I will try to make it work and maybe contribute with some documentation after that. 

Thank you and keep the good work!
